ui/src/app/views/navbar/navbar.component.ts

```diff
@@ -114,6 +125,22 @@ export class NavbarComponent implements OnInit, AfterViewInit {
             }
         });
 
+        // Listen change on recent projects viewed
+        this._projectStore.getRecentProjects().subscribe(projects => {
```

Comment:
Missing subscription
Check if there is AutoUnsubscribe on component

Reply:
done